### PR TITLE
BUG: re-add accidently removed flag.Parse

### PR DESCRIPTION
**Pull request by developer-guy**
### What this PR does / why we need it:

In the commit at https://github.com/fluent/fluent-operator/commit/d8e206004898e6e86f1bcbfeaf192916f9fb6293 I saw that someone accidentally removed the flag.Parse line, which caused fluent-watcher to ignore any CLI flag values, so I added it back

### Which issue(s) this PR fixes:

Fixes re

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by developer-guy** [comment]
kindly ping @benjaminhuo

File: cmd/fluent-watcher/fluentbit/main.go [hunk 1]
```diff
@@ -72,6 +72,8 @@ func main() {
 		level.Warn(logger).Log("--flb-timeout is deprecated. Consider setting the terminationGracePeriod field on the `(Cluster)FluentBit` instance.")
 	}
 
+	flag.Parse()
```
Comment (benjaminhuo) [inline cmd/fluent-watcher/fluentbit/main.go:75]: Nice catch @developer-guy 

cc @cw-Guo

**Review by benjaminhuo — APPROVED** [review]

**Comment by developer-guy** [comment]
Thanks @benjaminhuo, one question though, will you cut a new release with that fix included?

**Comment by developer-guy** [comment]
kindly ping @benjaminhuo

**Comment by cw-Guo** [comment]
@developer-guy A new version is scheduled, see this [issue](https://github.com/fluent/fluent-operator/pull/1286?notification_referrer_id=NT_kwDOAuKsq7QxMTgyNzQyNjcyMDo0ODQwOTc3MQ&notifications_query=is%3Adone#issuecomment-2284144689). thanks for the good catch again!

**Pull request merged**